crates/core_arch/src/aarch64/tme.rs

```diff
@@ -0,0 +1,124 @@
+//! ARM's Transactional Memory Extensions (TME).
+//!
+//! This CPU feature is available on Aarch64 – ARMv8-A arch. onwards.
+//! This feature is in the non-neon feature set. TME specific vendor documentation can
+//! be found [TME Intrinsics Introduction][tme_intrinsics_intro].
+//!
+//! The reference is [ACLE Q4 2019][acle_q4_2019_ref].
+//!
+//! ACLE has a section for TME extensions and state masks for aborts and failure codes.
+//! In addition to that [LLVM Aarch64 Intrinsics][llvm_aarch64_int] are
+//! self explanatory for what needs to be exported.
```

Comment:
This feels more like documentation of the implementation than documentation for the user of these intrinsics. Doc comments should try to be the latter.